src/components/CardLayout/CardLayout.js

```diff
@@ -7,6 +7,7 @@ import { Inside } from '../../utils/inside'
 function CardLayout({ children, columnWidthMin, rowHeight, ...props }) {
   const { layoutName } = useLayout()
   const fullWidth = layoutName === 'small'
+  const gridAutoRowValue = rowHeight === 'auto' ? rowHeight : `${rowHeight}px`
```

Comment:
Shouldn’t it be `auto`?

```suggestion
  const gridAutoRowValue = rowHeight === 'auto' ? 'auto' : `${rowHeight}px`
```

Reply:
Yes; my thought process was that if `rowHeight` is indeed equal to 'auto', then we should "leave it the same", instead of applying string interpolation. However, it is more clear to set it to auto.

Reply:
Oh right! Sorry for some reason I thought that it was using the default number alone rather of `auto` by looking at it. All good, let’s keep it the way you did it!